nexus/templatetags/nexus_helpers.py

```diff
@@ -21,11 +20,6 @@ def nexus_version():
     return nexus.__version__
 
 
-@register.simple_tag
-def nexus_csrf_cookie_name():
```

Comment:
I'd document the removal of this in HISTORY.rst just in case someone has used it in their own templates